src/cwbmg.c

```diff
@@ -13,11 +13,10 @@
  *  @author J. Ator
  *  @date 2005-11-29
  *
- *  @param[in] bmg     - character*(*): array containing BUFR
- *                        message to be written
- *  @param[in] nmb     - integer: size (in bytes) of BUFR message
+ *  @param[in] bmg     - char*: BUFR message to be written
+ *  @param[in] nmb     - f77int: Size (in bytes) of BUFR message
  *                        in bmg
- *  @param[out] iret    - integer: return code
+ *  @param[out] iret    - f77int: return code
```

Comment:
what is f77int?

Reply:
It's a typedef defined in bufrlib.h, equivalent to either an int or a long depending on whether it's a _4 or _8 build.

Reply:
Thanks!